installer/manifests/keptn/templates/api-gateway-nginx.yaml

```diff
@@ -383,6 +383,11 @@ spec:
     spec:
       {{- include "keptn.apiGatewayNginx.pod-security-context" . | nindent 6 }}
       terminationGracePeriodSeconds: {{ .Values.apiGatewayNginx.gracePeriod | default 60 }}
+      initContainers:
+        {{- include "keptn.initContainers.wait-for-nats" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-keptn-mongo" . | nindent 8 }}
```

Comment:
Generally speaking, only `shipyard-controller` and `mongo-datastore` connects directly to the DB. So I would have it only for these two services the wait for `keptn-mongo`

Reply:
done